src/Jira/Api.php

```diff
@@ -486,6 +514,45 @@ public function createVersion($projectId, $name, $options = array())
     }
 
     /**
+     * Update JIRA Version
+     *
+     * https://docs.atlassian.com/jira/REST/latest/#api/2/version-updateVersion
+     * 
+     * @param int $versionId Version identifier
+     * @param array $params Key->Value list to update the version with. 
+     *
+     * @return Result|false
+     */
+    public function updateVersion($versionId, $params = array()) {
```

Comment:
In this method yes, but you've added other methods as well and they have same problem.